index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>task - html-and-css</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+
+<body>
+
+    <header class="header" id="home">
+        <div class="wrapper">
+            <nav class="header__nav">
+                <div class="header__nav--logo">
+                    <img src="images/logo.svg" alt="">
+                </div>
+                <div class="header__nav--links">
+                    <a href="#team">The Team</a>
+                    <a href="#pricing">Pricing</a>
+                    <a href="#features">Features</a>
+                </div>
+            </nav>
+            <div class="header__content">
+                <div class="header__content--left">
+                    <h1 class="header__content--left-title">Beautiful Free Nova template</h1>
+                    <p class="header__content--left-txt">A top notch premium quality template for your next web project.
+                    </p>
+                    <button class="header__content--left-btn">Download FREE!</button>
+                </div>
+                <div class="header__content--right">
+                    <img class="header__content--right-img" src="images/screen.png" alt="website layout">
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <section class="features" id="features">
+        <h2 class="section_header features__header">features</h2>
```

Comment:
To nazwa BLOCKU czy ELEMENTU? Jeśli a) to `section-header`, jeśli b) to `section__header`